spec/project_spec.rb

```diff
@@ -0,0 +1,40 @@
+# spec/project_spec.rb
+require './lib/player'
+require './lib/board'
```

Comment:
`require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.